keystrokes/src/components/Character/CharacterDetails/CharacterDetailPage.js

```diff
@@ -77,6 +92,10 @@ function CharacterDetailPage({ userId, match }) {
 	async function selectCharacter() {
 		let updates = {};
 		updates[`/Users/${userId}/selectedCharacter`] = characterId;
+		updates[`/Characters/${characterId}/selectedSpec`] = spec;
```

Comment:
I forget does this do a batch so each update is individual or does this end up being one write?

Reply:
Im not 100% sure, the docs say `Simultaneous updates made this way are atomic: either all updates succeed or all updates fail.` so, i would guess its one write, since they are dependant on eachother being successful.